### ISSUE TITLE
Single Value, Multiple Keys/Classes

### ISSUE THREAD

**Issue by craigerskine** [tailwindcss/discuss#376]
Is there a way to have a single value generate into multiple keys/classes?

**Config**
```
module.exports = {
  theme: {
    colors: {
      gray: {
        [500, default]: '#CCC',
        ...
      }
    }
  }
}
```

**Output**
`.text-gray-500, .text-gray { color: #CCC; }`
...

**Comment by hacknug** [comment]
I don't think so. I think you're gonna have to add one key for each class you want to generate or write a custom function that handles this for you and then you spread the object this one returns 👍
